lib/runners/cplusplus.rb

```diff
@@ -0,0 +1,22 @@
+module Runners
+  module CPlusPlus
+#    FIELD_INCLUDE_PATH = { 'include-path': enum?(string, array(string)) }
+
+    def option_include_path
+      includes = Array(config_linter[:'include-path'] || find_paths_containing_headers)
+      includes.map { |v| "-I" + v }
+    end
```

Comment:
[suggestion]

- `config_include_path` seems better from `config_linter`.
- [nitpick] we can inline the `includes` variable.

```suggestion
    def config_include_path
      Array(config_linter[:'include-path'] || find_paths_containing_headers).map { |v| "-I#{v}" }
    end
```